src/BrightScriptDebugSession.ts

```diff
@@ -948,6 +959,59 @@ export class BrightScriptDebugSession extends DebugSession {
         await Promise.all(promises);
     }
 
+    /**
+     * Will search the project files for the comment "\' vs_code_tracker_entry" and replace it with the code needed to start the TrackerTask.
+     * @param stagingPath
+     */
+    public async injectTrackerTaskCode(stagingPath: string) {
```

Comment:
Can you add a unit test to validate this works?

Reply:
Tests added.